CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ This project <em>does not yet</em> adheres to [Semantic Versioning](https://semv
 - Image matrix link in Interactive keys task
 
 ### Changed
-- Made returning count from /controlled_vocabulary_terms optional # @jlpereira Potentially UI breaking check for use, and add &count=true to request if required
+- Plural params for identifiers API endpoint merged to array single form. e.g., identifier_object_ids[]=47&identifier_object_ids[]=2232 => identifier_object_id[]=47&identifier_object_id[]=2232. [#2195] 
```

Comment:
@jrflood re your question.  This line, and the associated header, should go in the PR comments.